### ISSUE TITLE
Missing shortcuts

### ISSUE THREAD

**Issue by qwench** [sandy081/vscode-todotasks#6]
Hi

Since the last update, I lost the Alt-D shortcut. I didn't find any references in todo shortcut in settings.
Is it possible to customize shortcuts ?

**Comment by sandy081** [comment]
Hey @qwench I will look into it and let you know. Thanks.

**Comment by sandy081** [comment]
@qwench All shortcuts are working properly.. Can you please provide more information like, what you are trying to do and which file etc.,

You can customize them by opening keybindings.json file. You can see the current/default shortcuts in Default Keyboard Shortcuts file.

**Comment by qwench** [comment]
I have updated to the latest version, and now when I press F1 todo, I see the 3 commands, but only the 'new task' has the ctrl-enter shortcut. Others commands have no shortcut displayed.
I tried alt-d to complete a task, but it doesn't work. I thing shortcuts have been removed.
If I open the 'préférences/keyboard shortcut', there are no 'todo' settings in the file.

I wil try to re-install the extension.
